test/studies/isx/isx-no-return.graph

```diff
@@ -0,0 +1,5 @@
+perfkeys: total =, bucket count =, bucket offset =, bucketize =, exchange =, count keys =
+files: isx-no-return.dat
```

Comment:
@benharsh's multi-locale tests were missing some lines which he guessed was due to this, though...